elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -346,6 +365,80 @@ public T getObject() {
     }
 
     @Override
+    public <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Optional<Expression> filterExpression
+    ) {
+        Object val = com.yahoo.elide.core.PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
Doesn't this method get called from `PersistentResource`? Would `getValue` be more appropriately placed in the `EntityDictionary` (assuming it's not "checked")?

Realistically, `getValue` is just reflection code that extracts the value from a POJO. We should probably isolate the notion of looking up values in beans and the security around being able to access that value anyway.